pkg/api/v20230904/openshiftcluster.go

```diff
@@ -64,6 +64,9 @@ type OpenShiftClusterProperties struct {
 	// The cluster worker profiles.
 	WorkerProfiles []WorkerProfile `json:"workerProfiles,omitempty"`
 
+	// The cluster worker profiles status.
+	WorkerProfilesStatus []WorkerProfile `json:"workerProfilesStatus,omitempty"`
```

Comment:
i'm wondering if we should make a new type for this.  In the update code, we never actually set some of the parameters that exist within a workerProfile struct.

Reply:
WorkerProfilesStatus - I see all the field are set/referenced under _worker_profile.go_ package while performing the enrichment.